openmc/deplete/results.py

```diff
@@ -485,7 +489,7 @@ def save(op, x, op_results, t, power, step_ind, proc_time=None):
         results.k = [(r.k.nominal_value, r.k.std_dev) for r in op_results]
         results.rates = [r.rates for r in op_results]
         results.time = t
-        results.power = power
+        results.source_rate = source_rate
```

Comment:
Right now, the `Results` class doesn't really have that information. It would have to either be passed from the `Integrator` (which also doesn't keep track of the units currently) or `Results` would have to dig into the `Operator` class to determine what units are being used. Either way doesn't seem great to me at present. I would probably lean toward just leaving it as is, but let me know if you see a cleaner way to do this.

Reply:
Yeah I'm good with leaving it as it is. As you mentioned in our offline convo, it isn't really possible to know everything from the current `depletion_results.h5` spec as it stands now.